Eatery/Network/NetworkManager.swift

```diff
@@ -94,14 +103,27 @@ struct NetworkManager {
         }
     }
 
-    func getBRBAccountInfo(sessionId: String, completion: @escaping (BRBAccount?, Error?) -> Void) {
+    func getBRBAccountInfo(sessionId: String, completion: @escaping (BRBAccount?, NetworkError?) -> Void) {
         apollo.fetch(query: BrbInfoQuery(accountId: sessionId)) { (result, error) in
-            if let error = error { completion(nil, error); return }
-            guard let result = result, let data = result.data, let accountInfo = data.accountInfo else { return }
+            guard error == nil else {
+                completion(nil, NetworkError(message: error?.localizedDescription ?? ""))
+                return
+            }
+
+            guard let result = result,
+                let data = result.data,
+                let accountInfo = data.accountInfo else {
+                    completion(nil, NetworkError(message: "could not safely unwrap response"))
```

Comment:
Is NetworkError something that is presented to the user?

Reply:
No